relayer/src/worker/wallet.rs

```diff
@@ -20,6 +20,9 @@ pub fn spawn_wallet_worker<Chain: ChainHandle>(chain: Chain) -> TaskHandle {
             TaskError::Ignore(format!("failed to query balance for the account: {e}"))
         })?;
 
+        // The input domain `balance.amount` may exceed u64::MAX.
+        // TODO: add mechanism workaround to handle `PosOverflow` errors in parse.
```

Comment:
@romac when you get a chance, can you have a look at how we can fix this TODO please?